cumulus/thin-egress.tf

```
@@ -35,12 +35,14 @@ resource "aws_secretsmanager_secret" "thin_egress_urs_creds" {
 resource "aws_secretsmanager_secret_version" "thin_egress_urs_creds" {
   secret_id = aws_secretsmanager_secret.thin_egress_urs_creds.id
   secret_string = jsonencode({
-    UrsId   = var.urs_client_id
-    UrsAuth = base64encode("${var.urs_client_id}:${var.urs_client_password}")
+    UrsId   = local.urs_tea_client_id
+    UrsAuth = base64encode("${local.urs_tea_client_id}:${local.urs_tea_client_password}")
   })
 }
 
 resource "aws_s3_object" "bucket_map_yaml" {
+  # If bucket_map_key is set, we already have a bucket map on S3 and don't need to create one
```

Maybe clarify that it's coming from the daac module instead something like this:

```suggestion
  # If bucket_map_key is set, the daac module already created one and we can skip creation here
```

---

Good call - updated on the next commit.